plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8420,6 +8420,14 @@ public void testAutoPurgeProperty()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testExplainAnalyzeTopLevelTimes()
```

Comment:
Since this is not affected by choice of connector, you can put this test in `AbstractTestEngineOnlyQueries` after all the existing EXPLAIN tests in that class.